<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmTool</class>
 <widget class="QWidget" name="frmTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <widget class="QTextEdit" name="txtMain">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>150</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>6</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="rightMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLineEdit" name="txtH264">
        <property name="text">
         <string>d:/1.mp4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenH264">
        <property name="text">
         <string>选择H264文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtAAC">
        <property name="text">
         <string>d:/1.aac</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenAAC">
        <property name="text">
         <string>选择AAC文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtMP4">
        <property name="text">
         <string>d:/out.mp4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpenMP4">
        <property name="text">
         <string>选择MP4文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="btnMp4Cmd">
           <property name="text">
            <string>命令方式转换</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnMp4Code">
           <property name="text">
            <string>代码方式转换</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cboxCmd">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCmd">
        <property name="text">
         <string>执行命令</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnClear">
        <property name="text">
         <string>清空数据</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtH264</tabstop>
  <tabstop>txtAAC</tabstop>
  <tabstop>txtMP4</tabstop>
  <tabstop>btnOpenH264</tabstop>
  <tabstop>btnOpenAAC</tabstop>
  <tabstop>btnOpenMP4</tabstop>
  <tabstop>txtMain</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
